viadot/tasks/azure_data_lake.py

```diff
@@ -360,6 +360,32 @@ def run(
         return df
 
 
+class AzureDataLakeFlattenDF(Task):
```

Comment:
This task is in no way connected to Azure, should be just `FlattenDataFrame` and in `utils.py`.